src/coreclr/dlls/mscoree/coreclr/CMakeLists.txt

```diff
@@ -98,19 +104,26 @@ set(CORECLR_LIBRARIES
     ceefgen
     comfloat_wks
     corguids
+    gcinfo
```

Comment:
@kg gcinfo has been moved into singlefilehost in https://github.com/dotnet/runtime/pull/113948. On Android with static linking, the app is linked against coreclr_static.a rather than singlefilehost, and coreclr_static.a still requires gcinfo.

What was the reason for moving gcinfo?

Reply:
Ok, I see it can't resolve external symbol `assertAbort`.